### PR TITLE
Gora 482 - Moving Gora to support only Hadoop2

### PR DESCRIPTION
**Pull request by renato2099**
Tests pass, but there are some extra hadoop dependencies that we don't seem to use any more, so they are commented out for now.

hadoop-common
hadoop-auth
hadoop-mapreduce-client-core
hadoop-yarn-common

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -909,11 +855,34 @@
...
       <dependency>
         <groupId>org.apache.hadoop</groupId>
-		    <artifactId>hadoop-client</artifactId>
-		    <version>${hadoop-2.version}</version>
-      </dependency>
+        <artifactId>hadoop-client</artifactId>
+        <version>${hadoop-2.version}</version>
+        <optional>true</optional>
+      </dependency>
+      <!--<dependency>-->
```
Comment (lewismc) [inline pom.xml]: Can you just remove this code rather than comment it out?

**Comment by lewismc** [comment]
Thanks @renato2099 for clearing this up. Keeping Gora more lightweight is the way forward now. As you've stated, not many people are using MRv1 API now.

**Comment by renato2099** [comment]
Done @lewismc, please let me know what you think.

**Comment by lewismc** [comment]
ACK, please merge into master and remove the modules.

**Comment by lewismc** [comment]
Dynamite

**Pull request merged**